docker-compose-config/monstache/config/scripts/transform_group.js

```diff
@@ -1,3 +1,3 @@
 module.exports = function(doc) {
-  return _.omit(doc, 'children', 'ancestors', 'isALeaf', 'hierarchy');
+  return doc;
```

Comment:
looks like a very complicated function... 😕 
You should at least add some lines of documentation here.

Reply:
you are probably right